features/step_definitions/authenticated_user_steps.rb

```diff
@@ -37,6 +37,23 @@
   visit page_path(path_name)
 end
 
+Given("I have incorrect credetialls") do
+  OmniAuth.config.mock_auth[:facebook] = :invalid_credentials
+end
+
+Given("I visit the site") do
+  visit root_path
+end
+
+Then("I should be redirected to the landing page") do
+  expect(page.current_path).to eq root_path
+end
```

Comment:
You already have the following step def:
```

Then("I should be on the {string} page") do \|path_name\|

Then("I should be on the {string} page") do \|path_name\|
  expect(page.current_path).to eq page_path(path_name)
end
```
Why not use that one here?

Reply:
Good point! I'll fix it!